camel/typing.py

```diff
@@ -28,6 +28,9 @@ class ModelType(Enum):
     GPT_4 = "gpt-4"
     GPT_4_32k = "gpt-4-32k"
     STUB = "stub"
+    # "Warning":the "text-davinci-003" model version is deprecated.
+    # Migrate before January 4, 2024.
+    TEXT_DAVINCI_003 = "text-davinci-003"
```

Comment:
Let's not add the deprecated model. Use whatever model is already there, they all support completion.

Reply:
As you mentioned it, I will try to move the `TEXT_DAVINCI_003`.

Reply:
Now I have removed this model.